packages/web/src/pages/_app.tsx

```diff
@@ -109,6 +111,9 @@ class NextApp extends App<AppInitialProps & WithApolloProps<{}>> {
       const { currency } = JSON.parse(settings)
       this.setState({ isCurrencyDEV: currency === 'DEV' })
     }
+
+    // Google Analytics
+    Router.events.on('routeChangeComplete', url => gtag.pageview(url))
```

Comment:
ルーティングされたら、gtag.pageviewをCallして欲しいので、routeChangeCompleteイベントにHookしています。これを、completeDidMountにHookしているのですが、もっと適切な場所はあったりしますかね？

Reply:
define and use `GoogleTagManager` components in Next.js examples.
https://github.com/vercel/next.js/blob/canary/examples/with-google-tag-manager/components/GoogleTagManager.js

But I don't think it's a problem, because this implementation is working fine 👍

Reply:
Thank you for the reference information. I'm going to upload the current program